Quetoo.vs15/src/config.h

```diff
@@ -0,0 +1,86 @@
+#ifndef __attribute__
+#define __attribute__(...)
+#endif
+
+#ifndef __thread
+#define __thread __declspec(thread)
+#endif
+
+#ifndef usleep
+#define usleep(t) Sleep(t / 1000)
+#endif
+
+#ifndef S_ISDIR
+#define S_ISDIR(m) (((m) & S_IFDIR) == S_IFDIR)
+#endif

```

Comment:
This is probably another case where, like in Objectively/MVC, we want to keep POSIX compat stuff outside of config.h.